pages/faq.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from "react";
+import { render } from "react-dom";
+import { faq_data } from "../utils/constants.js";
+
+export default class Faq extends Component {
+  render() {
+    return (
+      <div className="faq">
+        <style jsx>
+          {`
+            .faq {
+              display: flex;
+              justify-content: center;
+              flex-direction: column;
+              background-color: #3ec6ff;
+              font-family: "arial";
+              padding-bottom: 50px;
+            }
+            .heading {
+              display: flex;
+              justify-content: center;
+              align-items: center;
+              flex-direction: column;
+              height: 100%;
+              font-size: 2rem;
+              color: black;
+              font-weight: 600;
+              padding-top: 50px;
+            }
+            .question {
+              display: flex;
+              justify-content: center;
+              align-items: left;
+              flex-direction: column;
+              height: 100%;
+              font-size: 1.6rem;
+              color: black;
+              font-weight: 500;
+              padding-left: 50px;
+            }
+            .answer {
+              display: flex;
+              justify-content: center;
+              align-items: left;
+              flex-direction: column;
+              height: 100%;
+              font-size: 1.2rem;
+              color: white;
+              font-weight: 400;
+              padding-left: 100px;
+            }
+          `}
+        </style>
+        <p className="heading">FREQUENTLY ASKED QUESTIONS</p>
```

Comment:
**Map the array** and then display the question and answers in **separate components**.

Reply:
@ishukhanchi Try use Array.map() instead of using indexes to get array details.
[https://reactjs.org/docs/lists-and-keys.html](https://reactjs.org/docs/lists-and-keys.html)

Reply:
Done. @ankitjena